kratos/processes/variational_distance_calculation_process.h

```diff
@@ -185,10 +187,8 @@ class VariationalDistanceCalculationProcess : public Process
     /// Destructor.
     ~VariationalDistanceCalculationProcess() override
     {
-
-        Model& current_model = mr_base_model_part.GetModel();
```

Comment:
Here was the problem

Reply:
well, the model should be "the last to die"...

Reply:
problem is the mr_base_model_part not the model (see my other comment)